wx/lib/intctrl.py

```diff
@@ -860,7 +860,7 @@ def _fromGUI( self, value ):
                 return int( value )
             except ValueError:
                 if self.IsLongAllowed():
-                    return long( value )
+                    return LONGTYPE( value )
```

Comment:
By default `IsLongAllowed` returns `False` on Python3, unless the user has deliberatly used `SetLongAllowed(True)`. I'd rather behave as though the user knows what they're doing (with respect to Python3) then inhibit them entirely. 

For example, the user could have monkey wrenched a custom type in to `wx.lib.intctrl.LONGTYPE` and were expecting it to be used by using `SetLongAllowed(True)`

Reply:
Are you sure about that? L475 says the opposite.

Reply:
Also when for me `IsLongAllowed() == True` on Python3, that's also how i came across this.

Reply:
Oh, I skimmed over that too quickly. I was getting `0` by default under slightly different circumstances. I'm curios about the initial logic behind that `else` clause.  

I'll make a change that ensures that reusing `int` is avoided, that being said, I'm still in favor of enabling pythonic approaches rather than disabling them.